build/AzurePipelineTemplates/CsWinRT-BuildAndTest-Stage.yml

```diff
@@ -169,7 +169,8 @@ stages:
 
           set cswinrt_echo=on
           set cswinrt_build_only=true
-          set cswinrt_build_params=/bl:$(Build.SourcesDirectory)\cswinrt.binlog /p:CleanIntermediateDirs=true
+          rem AllowedReferenceRelatedFileExtensions includes all the default extensions other than pdb to address space issues resulting from pdbs being copied for each project reference.

```

Comment:
The comment is a bit confusing, it seems to suggest we are taking up more space by including more binaries. I like the description you have in this PR, we can reword this comment?

Reply:
Addressed.

Reply:
Addressed.